portal-impl/src/com/liferay/portal/tools/TCKtoJUnitConverter.java

```diff
@@ -135,7 +134,7 @@ private void _convert(String className, String message, File outputDir)
 		sb.append("\">\n");
 		sb.append("\t<properties>\n");
 
-		Properties properties = new SortedProperties(System.getProperties());
+		Properties properties = System.getProperties();
```

Comment:
@dantewang @ChiZelin I think we need order here. Without sorting, you will see the output is out of order.

Tina.